analyzers/its/IntegrationTests.shproj

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- This is a minimal Shared project to show directory structure in Solution Explorer. This project should not be build nor referenced. -->
+  <Import Project="$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\CodeSharing\Microsoft.CodeSharing.CSharp.targets" />
+  <ItemGroup>
+    <None Include="*.ps1" />
+    <None Include="*ruleset*" />
+    <None Include="*.targets" />
```

Comment:
Let's include just all of them, to avoid surprises/maintenance cost in the future. For example the `.gitignore` could also be useful to ahve.
```suggestion
    <None Include="*" />
```